modules/rosa-sharing-vpc-creation-and-sharing.adoc

```diff
@@ -10,35 +10,99 @@ You can share subnets within a configured VPC with another AWS user account if t
 ----
+$ cat <<EOF > /tmp/shared-vpc-policy.json
 {
-  "Version": "2012-10-17",
-  "Statement": [
-    {
-	  "Sid": "Statement1",
-	  "Effect": "Allow",
-	  "Principal": {
-	  	"AWS": "arn:aws:iam::<Account-ID>:root"
-	  }, <1>
-	  "Action": "sts:AssumeRole"
-	}
-  ]
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+                "route53:ChangeResourceRecordSets",
+                "route53:ListHostedZones",
+                "route53:ListHostedZonesByName",
+                "route53:ListResourceRecordSets",
+                "route53:ChangeTagsForResource",
+                "route53:GetAccountLimit",
+                "route53:GetChange",
+                "route53:GetHostedZone",
+                "route53:ListTagsForResource",
+                "route53:UpdateHostedZoneComment",
+                "tag:GetResources",
+                "tag:UntagResources"
+            ],
+            "Resource": "*"
+        }
+    ]
 }
+EOF
+----
++
+. Create the policy in AWS:
++
+[source,terminal]
+----
+$ aws iam create-policy \
+    --policy-name SharedVPCPolicy \
+    --policy-document file:///tmp/shared-vpc-policy.json
+----
++
+You will attach this policy to a role necessary for the shared VPC permissions.
++
+. Create a custom trust policy file that grants permission to assume roles:
++
+[source,terminal]
+----
+$ cat <<EOF > /tmp/shared-vpc-role.json
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Principal": {
+                "AWS": "arn:aws:iam::<Account-ID>:root"  <1>
+            },
+            "Action": "sts:AssumeRole"
+        }
+    ]
+}
+EOF
 ----
 +
 --
-<1> The following principals are be added later in this process after the *cluster-creating AWS account* user has created these roles. On creation, you must create a root user placeholder by using the *cluster-creator's AWS Account* ID as `arn:aws:iam::{Account}:root`.
+<1> The principal will be scoped down later in this process after the *cluster-creating AWS account* user has created the necessary cluster roles. On creation, you must create a root user placeholder by using the *cluster-creator's AWS Account* ID as `arn:aws:iam::{Account}:root`.
+--
++
+. Create the IAM role:
++
+[source,terminal]
+----
+$ aws iam create-role --role-name <role_name> \  <1>
+    --assume-role-policy-document file:///tmp/trust-policy.json
+----
++
+--
+<1> Replace _<role_name>_ with the name of the role you want to create.
+--
++
+. Attach the custom `SharedVPCPolicy` permissions policy:
++
+[source, terminal]
+----
+$ aws iam attach-role-policy --role-name <role_name> --policy-arn \  <1>
+    arn:aws:iam::<AWS_account_ID>:policy/SharedVPCPolicy  <2>
+----
++
+--
+<1> Replace _<role_name>_ with the name of the role you created.
+<2> Replace _<AWS_account_ID>_ with the *VPC-owning AWS account* ID.
 --
-    * The `ResourceGroupandTagEditorFullAccess` permissions policy
```

Comment:
What about this set of permissions? we attached it as well in previous installations.

Reply:
> What about this set of permissions? we attached it as well in previous installations.

@wgordon17 Can you please advise on this? Thanks.

Reply:
@oriAdler @AedinC This permissions policy goes away per @jharrington22. We can't be relying on non-RH-managed policies. The _important_ permissions are apparently handled with https://github.com/openshift/openshift-docs/pull/64578/files#diff-4fcef1c9804545adc4f3564dccb0a0a927043c96c1a28e7253e145a55b2c0bc4R36-R37

So we're all good there 👍